bootstrap-scripts/package_install.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -v
+
+set -ex
+if [ '$pnda_mirror$' != '$''pnda_mirror''$' ]; then 
+  PNDA_MIRROR=$pnda_mirror$
+fi
+DISTRO=$(cat /etc/*-release|grep ^ID\=|awk -F\= {'print $2'}|sed s/\"//g)
+if [ "x$DISTRO" == "xubuntu" ]; then
+  export DEBIAN_FRONTEND=noninteractive
+  if [ "x$PNDA_MIRROR" != "x" ]; then
```

Comment:
How about asserting this instead and error handling (with a clear message before exit)?